app/code/Magento/Inventory/Model/PackageInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+interface PackageInterface
+{
+    /**
+     * @return \Magento\InventoryApi\Api\Data\SourceInterface
+     */
+    public function getSource();
+
+    /**
+     * @return \Magento\Quote\Model\Quote\Item[]
+     */
+    public function getItems();
+
+    /**
+     * @return \Magento\Quote\Model\Quote\Address
+     */
+    public function getAddress();
+
+    /**
+     * @param $address \Magento\Quote\Model\Quote\Address
+     * @return $this
+     */
+    public function setAddress($address);
+
+    public function getQty();
```

Comment:
yes, because qty here is the number of the product in this particular package. It can be less or equal to the qty of the product in the cart. I.e. 10 products with sku1 were requested, 6 fulfilled from source 1, 4 from source 2.